components/combobox/__tests__/__snapshots__/combobox.snapshot-test.jsx.snap

```diff
@@ -360,6 +362,113 @@ exports[`Base Custom Menu Item Open HTML Snapshot 1`] = `
 </div>"
 `;
 
+exports[`Base Label Required DOM Snapshot 1`] = `
+<div
+  className="slds-form-element"
+>
+  <label
+    className="slds-form-element__label"
+    htmlFor="combobox-unique-id"
+  >
+    <abbr
+      className="slds-required"
+      title="required"
+    >
+      *
+    </abbr>
+    Search
+  </label>
+  <div
+    className="slds-form-element__control"
+  >
+    <div
+      className="slds-combobox_container"
+    >
+      <div
+        aria-describedby="description-unique-id-error-message"
+        aria-expanded={false}
+        aria-haspopup="listbox"
+        aria-owns="combobox-unique-id-listbox"
+        className="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click ignore-react-onclickoutside slds-has-error"
+        role="combobox"
+      >
+        <div
+          className="slds-combobox__form-element slds-input-has-icon slds-input-has-icon_right"
+          role="none"
+        >
+          <input
+            aria-activedescendant={null}
+            aria-autocomplete="list"
+            aria-controls="combobox-unique-id-listbox"
+            aria-describedby={undefined}
+            aria-expanded={undefined}
+            aria-labelledby={undefined}
+            aria-owns={undefined}
+            aria-required={undefined}
+            autoComplete="off"
+            className="slds-input slds-combobox__input"
+            disabled={undefined}
+            id="combobox-unique-id"
+            maxLength={undefined}
+            minLength={undefined}
+            name={undefined}
+            onBlur={[Function]}
+            onChange={[Function]}
+            onClick={[Function]}
+            onFocus={[Function]}
+            onInput={undefined}
+            onInvalid={undefined}
+            onKeyDown={[Function]}
+            onKeyPress={undefined}
+            onKeyUp={undefined}
+            onSelect={undefined}
+            onSubmit={undefined}
+            placeholder="Search Salesforce"
+            readOnly={false}
+            required={undefined}
```

Comment:
This needs to be true for accessibility.

See Input component:
![screen shot 2018-03-12 at 5 16 16 pm](https://user-images.githubusercontent.com/1290832/37314382-63bb256a-2619-11e8-8a1c-4160ba54fca9.png)